lib/muffin_man/fulfillment_outbound/v20200701.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module MuffinMan
+  module FulfillmentOutbound
+    class V20200701 < SpApiClient
+      # To get outbound fulfillment preview
+      # @param [MuffinMan::RequestHelpers::OutboundFulFillment::FulfillmentPreviewRequest] fulfillment_preview_request in the form of object
+      def get_fulfillment_preview(fulfillment_preview_request)
+        return unprocessable_entity(fulfillment_preview_request&.errors) unless fulfillment_preview_request&.valid?
+
+        @local_var_path = "/fba/outbound/2020-07-01/fulfillmentOrders/preview"
+        @request_body = fulfillment_preview_request.to_camelize
+        @request_type = "POST"
+        call_api
+      end
+
+      # To create outbound fulfillment order
```

Comment:
May be we don't need this comment as the method name is self indicative? Similarly we can remove it from other places that indicates what the method name is already indicating.

Reply:
I was trying to follow ruby doc([YARD](https://yardoc.org/)) format here!